spec/serializers/admin/enterprise_serializer_spec.rb

```diff
@@ -1,9 +1,61 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe Api::Admin::EnterpriseSerializer do
   let(:enterprise) { create(:distributor_enterprise) }
   it "serializes an enterprise" do
     serializer = Api::Admin::EnterpriseSerializer.new enterprise 
     serializer.to_json.should match enterprise.name  
   end
+
+  context "for logo" do
+    let(:enterprise) { create(:distributor_enterprise, logo: image) }
+
+    context "when there is a logo" do
+      let(:image) do
+        image_path = File.open(Rails.root.join("app", "assets", "images", "logo-black.png"))
+        Rack::Test::UploadedFile.new(image_path, "image/png")
+      end
+
+      it "includes URLs of image versions" do
+        serializer = Api::Admin::EnterpriseSerializer.new(enterprise)
+        expect(serializer.as_json[:logo]).to_not be_blank
```

Comment:
this line seems redundant to me, given the line below.

Reply:
Its main purpose is to test that the JSON provides logo information, before testing that it includes the thumbnail URLs. But this also makes sure that the example fails with "`serializer.as_json[:logo]` is not blank" if it is `nil`, not a missing method error in the next assertion.

Sounds reasonable enough?